recipes/soapysdr-module-airspyhf/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ex
+
+mkdir build
+cd build
+
+cmake_config_args=(
+    -DCMAKE_BUILD_TYPE=Release
+    -DCMAKE_INSTALL_LIBDIR=lib
+    -DCMAKE_INSTALL_PREFIX=$PREFIX
+)
+
+cmake ${CMAKE_ARGS} .. "${cmake_config_args[@]}"
```

Comment:
No need for the long list here, only one of them is not part of `CMAKE_ARGS`
```suggestion
cmake ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release ..
```